modules/sony/display/cbx.rb

```diff
@@ -0,0 +1,61 @@
+module Sony; end
+module Sony::Display; end
+
+# Documentation: https://docs.google.com/spreadsheets/d/1F8RyaDqLYlKBT7fHJ_PT7yHwH3vVMZue4zUc5McJVYM/edit?usp=sharing
+
+class Sony::Display::CBX
+    include ::Orchestrator::Constants
+    include ::Orchestrator::Transcoder
+
+    # Discovery Information
+    tcp_port 4999
+    descriptive_name 'Sony CBX Display'
+    generic_name :Display
+
+    def on_load
+        on_update
+    end
+
+    def on_update
+    end
+
+    def connected
+        schedule.every('60s') { power? }
+    end
+
+    def disconnected
+        schedule.clear
+    end
+
+    #
+    # Power commands
+    #
+    def power(state, opt = nil)
```

Comment:
`opt` is not used. This can either be removed as an argument, or, if you like to provide the ability to pass options through to accompany the outgoing command you can splat this into the send method.
```
send("8C 00 00 02 01 8F", hex_string: true, **opt)
```
If you do this, set it's default value to be an empty has rather than `nil` too.